modules/crypto/src/lib/crypto/crypto-hash-transform.js

```diff
@@ -21,12 +21,11 @@ export default class CryptoHashTransform {
       .toString(CryptoJS.enc.Base64);
   }
 
-  constructor(options) {
-    options = options || {};
-    options.crypto = options.crypto || {};
+  constructor(options = {}) {
+    options = {crypto: {}, modules: {}, ...options};
```

Comment:
Why add nesting, if modules are never used beyond the constructor, and you will always access the `options.crypto` for everything?

Reply:
`options.modules` is a generic mechanism across loaders.gl to provide big external modules, used e.g. for Draco3D. I am just aligning with existing practices. The modules option is documented here https://loaders.gl/modules/core/docs/api-reference/parse#options

Also note that this crypto option object is mixed with all other options, for loaders core function, for all other loaders (json, csv, etc in your code).

We already use nesting for loaders to keep things clean and conflict free so it makes sense to also use a namespace here. I suppose it could be a namespace per transform, but I am using `crypto` as a category namespace.